From 0f7415dd520e7f005a0fb42ce42975dca46ef8e8 Mon Sep 17 00:00:00 2001
From: Vagrant Cascadian <vagrant@reproducible-builds.org>
Date: Fri, 25 Dec 2020 08:39:37 +0000
Subject: [PATCH 1/2] Remove build date from autoconf.texi.

autoconf.texi is used to generate .pdf, .ps, .info and .html
documentation shipped with the package.

See "Timestamps are best avoided":

  https://reproducible-builds.org/docs/timestamps/
---
 doc/autoconf.texi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/doc/autoconf.texi b/doc/autoconf.texi
index 436c987..acdcd76 100644
--- a/doc/autoconf.texi
+++ b/doc/autoconf.texi
@@ -197,7 +197,7 @@
 
 @copying
 
-This manual (@value{UPDATED}) is for GNU Autoconf
+This manual is for GNU Autoconf
 (version @value{VERSION}),
 a package for creating scripts to configure source code packages using
 templates and an M4 macro package.
@@ -245,7 +245,7 @@ entitled ``GNU Free Documentation License.''
 @titlepage
 @title Autoconf
 @subtitle Creating Automatic Configuration Scripts
-@subtitle for version @value{VERSION}, @value{UPDATED}
+@subtitle for version @value{VERSION}
 @author David MacKenzie
 @author Ben Elliston
 @author Akim Demaille
-- 
2.20.1

